ipn/local.go

```diff
@@ -478,8 +482,6 @@ func (b *LocalBackend) readPoller() {
 
 		b.mu.Lock()
 		if b.hiCache == nil {
-			// TODO(bradfitz): it's a little weird that this port poller
-			// is started (by NewLocalBackend) before the Start call.
 			b.hiCache = new(tailcfg.Hostinfo)
```

Comment:
can you maybe document the `hiCache` field up where it's declared in the struct and say what it is?